src/poc/ansible/host_vars/srv1.yml

```diff
@@ -0,0 +1,45 @@
+bind_statistics_channels: true
+bind_statistics_allow: ['any']
+bind_listen_ipv4: ['any']
+bind_listen_ipv6: ['any']
+bind_allow_query: ['any']
+bind_recursion: true
+bind_allow_recursion: ['any']
+bind_forward_only: false
+bind_forwarders:
+  - 10.0.2.3
+  - 8.8.8.8
+  - 1.1.1.1
+bind_dnssec_enable: false
+bind_dnssec_validation: false
+bind_query_log: 'data/query.log'
+bind_zones:
+  - name: poc.lan
+    type: primary
+    primaries:
+      - 172.16.128.1
+    name_servers:
+      - srv1.poc.lan.
+    hosts:
+      - name: srv1
+        ip: 172.16.128.1
+        aliases:
+          - ns
+      - name: srv2
+        ip: 172.16.128.2
+        aliases:
+          - web
+      - name: srv3
+        ip: 172.16.128.3
+        aliases:
+          - grafana
+      - name: srv4
+        ip: 172.16.128.4
+        aliases:
+          - db
+      - name: srv5
+        ip: 172.16.128.5
+        aliases:
+          - base
```

Comment:
This doesn't match up with the table in your comment. I guess the table is out of date?

srv3 and srv4 are flipped around in the table (or here). Is srv3/4 running prometheus and grafana? In that case ignore my comment on srv5 potentially being a grafana node.

If that's the case maybe srv5 can be:

- A log collector (also pretty common)
  - Loki
  - Graylog
  - Logstash
  - Rsyslog
  - ...
- A haproxy node (reverse proxy, loadbalancing), also quite common
  - ELK stack

Reply:
Currently srv4 only runs Prometheus and Node Exporter. I could add Grafana to it, but I considering installing it on srv5 paired with a samba share. I just have to find up-to-date roles to help me with this.

See: 2059dff901ab57393858c2da116855b444d568d0